IMPORTING-DATA.md

```diff
@@ -221,14 +221,19 @@ Arrange to have the file you just created uploaded to the
 `govuk-custom-formats-mapit-storage-production` S3 bucket and ensure that
 it's permission is set to `public`.
 
-Once it's been uploaded change the URL and checksum (using
+### Update servers with new database
+
+**NB: THIS REQUIRES ACCESS TO GOV.UK PRODUCTION**
+
+Once the data has been uploaded change the URL and checksum (using
 `sha1sum <your-mapit-file.sql.gz>`) reference in `import-db-from-s3.sh`
 to refer to your new file. Submit change as a PR against
-[Mapit](https://github.com/alphagov/mapit) and deploy once it's been approved.
-
-### Update servers with new database
+[Mapit](https://github.com/alphagov/mapit) and deploy following the normal
+process.
 
-NB: THIS REQUIRES ACCESS TO GOV.UK PRODUCTION
+**Note: Only deploy this change to production once the new data has been tested
+in staging. If a new Mapit machine gets created in AWS, it will automatically
+try importing the data.**
```

Comment:
Are you sure this is the case? I assume the data is in a RDS database or something, so it is independent of the machines running Mapit?

Reply:
Each `mapit` machine [has its own database](https://github.com/alphagov/govuk-puppet/blob/bd5bfecce64a80e4fcac851ece2f03184355cf3a/modules/govuk/manifests/apps/mapit.pp#L42-L47), and bringing up a new machine [populates that database](https://github.com/alphagov/govuk-puppet/commit/fa75e912e0af3192ee803f158cb8130718f35b51). I think we did it like that so we can update each mapit instance individually without having to take them all down at once...

Reply:
Right, OK, I guess that makes sense :+1: